serviceinsight/license.md

```diff
@@ -27,6 +27,10 @@ The license can be loaded by clicking the "Browse..." button and selecting the l
 
 ![trial period licensed](images/trial-period-licensed.png)
 
+#### Imported locations
+
+The license is imported *at the user level * in the registry (`HKCU\SOFTWARE\ParticularSoftware\@License`) and since 1.8.0 also on the file system (`%LOCALAPPDATA%\ParticularSoftware\license.xml`).
```

Comment:
Same question about formatting.

Also, it's consistent with the other two: 
- "at" being included here
- space at the end

Reply:
Now aligned the formatting.